dataprofiler/profilers/numerical_column_stats.py

```diff
@@ -789,7 +789,7 @@ def _merge_biased_skewness(
         third_term = 3 * delta * (match_count1 * M2_2 - match_count2 * M2_1) / N
         M3 = first_term + second_term + third_term
 
-        biased_skewness = np.sqrt(N) * M3 / np.sqrt(M2**3)
+        biased_skewness: float = np.sqrt(N) * M3 / np.sqrt(M2**3)
```

Comment:
is this a float or is it a np.float64?

Reply:
@Sanketh7

Reply:
This does seem to be a np.float64. Because biased_skewness could be either `np.nan` or some `np.float64`, I've updated any usage of biased skewness to be a `Union[float, np.float64]`.